Project architecture/Mobile app architecture.md

```diff
@@ -147,7 +149,7 @@ class MachineFriendlyNameRepository @Inject constructor(
     private val getMachineInfo: Interactors.GetMachineInfo,
     private val setMachineFriendlyName: Interactors.SetMachineFriendlyName,
     private val getStoredMachineName: Interactors.GetStoredMachineName,
-    private val storeMachineName: Interactors.StoreMachineName
+    private val storeMachineName: Interactors.StoreMachineName,
     private val machineInfoToMachineNameMapper: Mappers.MachineInfoToMachineName
```

Comment:
Not a big deal, but the mapper in this constructor has `...Mapper` suffix in the name while interactors don't. For consistency it would be nice to stick to one way of naming thing, whatever way that is.